nan 2.3.1 broke compilation of sse4_crc32 on node v0.10.42 (Darwin)

ref: https://github.com/nodejs/nan/issues/562

Compilation/build of sse4_crc32@5.0.0 fails with the following errors:

```
../src/crc32c.cpp:240:5: error: no matching function for call to 'SetMethod'
    Nan::SetMethod(exports, "isHardwareCrcSupported", isHardwareCrcSupported);
    ^~~~~~~~~~~~~~
../../nan/nan.h:1853:17: note: candidate function not viable: no known conversion from 'Handle<v8::Object>' to
      'v8::Local<v8::Object>' for 1st argument
NAN_INLINE void SetMethod(
                ^
../../nan/nan.h:1865:17: note: candidate function not viable: no known conversion from 'Handle<v8::Object>' to
      'v8::Local<v8::FunctionTemplate>' for 1st argument
NAN_INLINE void SetMethod(
                ^
../../nan/nan.h:1876:17: note: candidate function not viable: no known conversion from 'Handle<v8::Object>' to
      'v8::Local<v8::ObjectTemplate>' for 1st argument
NAN_INLINE void SetMethod(
                ^
../src/crc32c.cpp:241:5: error: no matching function for call to 'SetMethod'
    Nan::SetMethod(exports, "calculateCrc", calculateCrc);
```

Works with nan 2.3.0 (via forced shrinkwrap). Maybe nan 2.3.1 should not have been a patch release?

--- a/src/crc32c.cpp
+++ b/src/crc32c.cpp
@@ -234,7 +234,7 @@ NAN_METHOD(calculateCrc) {
 /**
  * Initialize the module
  */
-void init(Handle<Object> exports) {
+void init(Local<Object> exports) {
     initCrcTable();
 
     Nan::SetMethod(exports, "isHardwareCrcSupported", isHardwareCrcSupported);

